webbpsf/opds.py

```diff
@@ -2443,18 +2455,14 @@ def random_unstack(ote, radius=1, verbose=False):
 
 # --------------------------------------------------------------------------------
 
-def segment_primary(infile='JWpupil.fits'):
+def segment_primary(infile='JWpupil.fits', npix=1024):
```

Comment:
This segment_primary function is old I think not used anymore. 

This has been superseded by `webbpsf.optics.WebbPrimaryAperture()` which can be called with `label_segments=True`.

I think this whole function can just be deleted. (I'd forgotten it was here... Gotta admit it's strange to now be finding code I've written more than a decade ago. 🤦)

Reply:
@kjbrooks First, thanks for working on this! Did you see my comment on this PR earlier today?  Apologies if not.  The `segment_primary` function doesn't need to be updated to work for larger sizes, since there's another better way to set up the segment map using webbpsf.optics.WebbPrimaryAperture()

Reply:
Oops.

Reply:
This is what happens when I get into a coding mode! I will go ahead and stop worrying about this then!!